packages/eslint/default.yml

```diff
@@ -265,3 +265,4 @@ rules:
 
   # Import rules (https://github.com/import-js/eslint-plugin-import)
   import/no-extraneous-dependencies: 2
+  import/no-unresolved: [ "error", { ignore: [ "^@yoast/(analysis-report|components|helpers|replacement-variable-editor|search-metadata-previews|social-metadata-forms|style-guide)" ] } ]
```

Comment:
As mentioned, I think these should go into the individual configurations, not in our preset.
Something like:
[no-unresolved.patch](https://github.com/Yoast/wordpress-seo/files/14445499/no-unresolved.patch)

Reply:
Absolutely right! I've added the package-specific config to each of my commits.